go.mod

```diff
@@ -16,5 +17,6 @@ require (
 	github.com/k0kubun/colorstring v0.0.0-20150214042306-9440f1994b88 // indirect
 	github.com/mattn/go-colorable v0.0.6 // indirect
 	github.com/mattn/go-isatty v0.0.0-20160806122752-66b8e73f3f5c // indirect
+	github.com/sergi/go-diff v1.2.0 // indirect
```

Comment:
I'm just wondering: why is this still here? Shouldn't we get rid of all the references of `github.com/sergi/go-diff` in favor of `github.com/Kong/go-diff`?